### PR TITLE
INGK-873 Set the RPDOMap values by byte string

### PR DESCRIPTION
**Pull request by MartinALL91**
### Description

Set the RPDOMap values by byte string. 

Fixes # INGK-873

### Type of change

- [x] Move the set_item_bytes method to the base PDOMap class.
- [x] Add tests.

### Tests
- [x] Add new unit tests if it applies.
- [x] Run tests.

### Documentation

Please update the documentation.

- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingenialink tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### Others

- [x] Set fix version field in the Jira issue.

### CODE REVIEW

File: ingenialink/pdo.py [hunk 1]
```diff
@@ -310,6 +310,28 @@ def items_mapping(self) -> bytearray:
             map_bytes += pdo_map_item.register_mapping
         return map_bytes
 
+    def set_item_bytes(self, data_bytes: bytes) -> None:
+        """Set the items raw data from a byte array received from the slave.
```
Comment (GullinBustin) [inline ingenialink/pdo.py:314]: Remove `received from the slave`, it is not a must.

File: ingenialink/pdo.py [hunk 2]
```diff
@@ -310,6 +310,28 @@ def items_mapping(self) -> bytearray:
...
+                f" obtained {len(data_bytes)}"
+            )
+        data_bits = bitarray.bitarray(endian=BIT_ENDIAN)
+        data_bits.frombytes(data_bytes)
+
+        offset = 0
+        for item in self.items:
+            item.raw_data_bits = data_bits[offset : item.size_bits + offset]
+            offset += item.size_bits
+
```
Comment (GullinBustin) [inline ingenialink/pdo.py:334]: I think that `get_item_bits` and `get_item_bytes` should be in the base class too

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: ingenialink/pdo.py (hunk 1 above)
Comment (MartinALL91) [reply ingenialink/pdo.py]: Done

File: ingenialink/pdo.py (hunk 2 above)
Comment (MartinALL91) [reply ingenialink/pdo.py:334]: Yes, It could be useful for the FSOE master.

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**